pennylane/math/__init__.py

```diff
@@ -23,49 +23,45 @@
     Furthermore, the names and behaviour of these functions may differ from similar
     functions in common frameworks; please refer to the function docstrings for more details.
 
-Internally, these functions dispatch by using the :class:`TensorBox` class, a container and API for
-array-like objects that allows array manipulation to be performed in a unified manner for supported
-tensor/array manipulation frameworks.
-
 The following frameworks are currently supported:
 
 * NumPy
 * Autograd
 * TensorFlow
 * PyTorch
+* JAX
 """
+import autoray as ar
+
 from .fn import (
-    T,
-    abs_ as abs,
```

Comment:
All the deleted functions here (plus many, many others) are automagically supported by Autoray, so we no longer need to implement them manually.

Using autoray as a backend, only two cases require us to manually implement functions:

- The function requires multiple dispatch (autoray only supports single dispatch)
- The function has implementations that differ between the frameworks.